.github/workflows/add-new-pr-to-oss-triaging.yml

```diff
@@ -12,7 +12,7 @@ jobs:
     name: Add external label to pull request if outside StackRox
     runs-on: ubuntu-latest
     env:
-      GH_TOKEN: ${{ github.token }}
+      GH_TOKEN: ${{ secrets.RHACS_BOT_GITHUB_TOKEN }}
```

Comment:
Will external contributors have access to the secrets? Not sure they will, right?

Reply:
They won't and this part of the workflow didn't actually fail in the example linked in the description.
I would suggest to keep it as is. The default `github.token` has sufficient privileges to add the label.

Reply:
So maybe we need to change `pull_request_target:` to `pull_request:`